docs/sources/plugins/developing/backend-plugins-guide.md

```diff
@@ -111,7 +111,7 @@ type JsonDatasource struct {
 }
 ```
 
-The only requirement for the plugin type is that it should extend `plugin.NetRPCUnsupportedPlugin`. You can include more fields into your struct if you want to add some datasource-specific features, like logging, cache etc:
+The only requirement for the plugin type is that it should extend `plugin.NetRPCUnsupportedPlugin`. You can include more fields into your struct if you want to add some data-source-specific features, like logging, cache etc:
```

Comment:
I'm not sure what the correct practice for this case. Would it be more correct to write `data source-specific` here?